### DISCUSSION TITLE
Only Supports 3 Email Accounts???

### DISCUSSION THREAD

**Discussion by itsbaraa** [bkaankose/Wino-Mail#377] [Q&A] [answered]
Does Wino-Mail support unified inbox?
I saw another discussion 6 months ago and this feature was still in beta is it released yet?

Edit: poor README.md, the README should clearly state that wino only supports 3 emails for free.

**Comment by bkaankose** [comment]
It's released under the name of "Linked Accounts" in Manage Accounts menu couple versions back. It's available in the Store version already.

**Comment by itsbaraa** [comment]
bruh.
![image](https://github.com/user-attachments/assets/90c46a08-0079-49ef-95d7-71140f787c74)
why are you even on github?
at least warn me that I'm only allowed to use 3 accounts before I install it.
I'm not gonna pay 10$ for a copy-paste UI with no extra features of the old outlook.
and you even ask for donations LOL

**Comment by bkaankose** [comment, marked as answer]
It's on the Microsoft Store overview and Donate section of the readme. Plus, you can literally build from the source code to revoke this.
I share free promo codes of this add-on with the people who actually contribute to the project's growth instead of being a clown like you. 

The project itself is on GitHub to collaborate with others. From now on I am myself on GitHub to deal with free rider parasites.

take it or leave it, bruh...
